src/DevTools/mockReactI18n.tsx

```diff
@@ -0,0 +1,14 @@
+import { get } from "lodash"
+
+const translations = require("System/i18n/locales/en-US/translation.json")
+
+export const useTranslation = () => {
+  return {
+    t: (path: string | string[]) => {
+      const key = Array.isArray(path) ? path.join(".") : path
+      const value = get(translations, key, key)
+
+      return value
+    },
+  }
+}
```

Comment:
cc @damassi

Reply:
OK looks like I'll just have to inject the i18n `t` function here directly, preloaded with the en file. Because we use some of the other 'features' (like pluralization).